guide/13-managing-arcgis-applications/hub-premium-guide.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<p>An Initiative supports policy or activity oriented goals through workflows, tools, and team collaboration. Initiatives manage the team, events, content, and experiences involved in achieving a shared goal.</p><p>You can add a new initiative with the following commands:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>